.golangci.yml

```diff
@@ -16,6 +16,8 @@ linters:
     - scopelint
     - varcheck
     - godox
+    - funlen
```

Comment:
I thought we didn't want to disable `funlen`? cc: @alexanderbez

Reply:
funlen is a new linter that wasnt present till this version `1.19.x` this will require quite a bit of refactoring if we use the default length.

Reply:
yeah, I noticed. Quite annoying